biolink-model.yaml

```diff
@@ -6220,6 +6223,36 @@ slots:
       - value: computational_model
       - value: text_mining_agent
 
+
+  process context qualifier:
+    is_a: qualifier
+    description: >-
+        A context qualifier representing the process in which an association takes place. 
+        The core association should form a part of the process, or describe how a process affects a state variable 
+        (in which case this field becomes redundant and can be omitted).
+    examples:
+      - value: ENVO:02500031 #Hydrological process
+      - value: ENVO:01001824 #Ecological succession
+    in_subset:
+      - ecolink
+
+  conditioned by qualifier:
```

Comment:
could you please add some examples here - I am not quite sure I fully understand what "conditioned by" means?

- can it have a value of [`upregulates`, `downregulates`] in one record?  
- how does it relate to the direction qualifier suite (the enumeration for this qualifier has values `upregulated` and `downregulated`).
- how does it relate to the `regulates` predicate?

Maybe the necessity of this slot means we should think about breaking the association into two edges?  I'm not sure; maybe an example would help?